tests/rise/rewrite.rs

```diff
@@ -221,36 +221,36 @@ fn separate_dot_hv_simplified() -> Rewrite<Rise> {
 }
 
 // subst using extraction
-fn beta_extr() -> Rewrite<Rise> {
-    let pat = Pattern::parse("(app (lam $1 ?b) ?t)").unwrap();
-    let s = Slot::numeric(1);
-
-    let a = pat.clone();
-    let a2 = pat.clone();
-
-    let rt: RewriteT<Rise, (), Vec<(Subst, RecExpr<Rise>)>> = RewriteT {
-        searcher: Box::new(move |eg| {
-            let extractor = Extractor::<_, AstSize>::new(eg, AstSize);
-
-            let mut out: Vec<(Subst, RecExpr<Rise>)> = Vec::new();
-            for subst in ematch_all(eg, &a) {
-                let b = extractor.extract(&subst["b"], eg);
-                let t = extractor.extract(&subst["t"], eg);
-                let res = re_subst(s, b, &t);
-                out.push((subst, res));
-            }
-            out
-        }),
-        applier: Box::new(move |substs, eg| {
-            for (subst, res) in substs {
-                let orig = pattern_subst(eg, &pat, &subst);
-                let res = eg.add_expr(res);
-                eg.union_justified(&orig, &res, Some("beta-expr".to_string()));
-            }
-        }),
-    };
-    rt.into()
-}
+// fn beta_extr() -> Rewrite<Rise> {
```

Comment:
I would leave this in using `#[allow(unused)]`. We might come back to use this some day.